utils/check_repo.py

```diff
@@ -88,6 +88,7 @@
 IGNORE_NON_TESTED = PRIVATE_MODELS.copy() + [
     # models to ignore for not tested
     "RecurrentGemmaModel",  # Building part of bigger (tested) model.
+    "ChineseCLIPTextModel",  # Building block of a bigger (tested) model.
```

Comment:
This is funny - why don't we need to add this for CLIPVisionModel?

Reply:
Ah, yes :laughing:  I don't think that's needed anymore. All this is from an initial design confusion between `...Model` and `...Transformer` in this model. I think the simplest thing to do is to exclude the new `ChineseCLIPTextTransformer` from checks, and make it inherit from `PretrainedModel` because of common checks in the forward. I can also rework the structure to have `...Model` in the multimodal model rather than `...Transformer` but that's a bit more work